movo_common/movo_ros/src/movo/README.md

```diff
@@ -0,0 +1,118 @@
+<h1>Movo Xbox teleoperation</h1>
+
+To use the Xbox 360 wireless controller instead of the logitech controller, install the Xbox driver:
+
+`sudo apt-get install xboxdrv`
+
+You should plug a wireless controller receiver in Movo or your computer.
+
+If you want to control the Movo with a logitech controller or with an Xbox controller, you should go in movo_common/movo_config/movo_config.bash and change 
+MOVO_JOY_IS_XBOX or MOVO_JOY_IS_LOGITECH. Put the joystick you want to true and the other to false. Do a `sync_robot -nc` and reboot Movo for the change
```

Comment:
Instead of the command, I would advise the user to recompile both catkin_ws on movo1 and movo2 and then reboot